packages/core/android/src/main/kotlin/com/datadog/reactnative/DdLogs.kt

```diff
@@ -84,4 +85,120 @@ class DdLogs(reactContext: ReactApplicationContext, logger: Logger? = null) :
         )
         promise.resolve(null)
     }
+
+    /**
+     * Send a log containing an error with level debug.
```

Comment:
minor grammar (same for other methods):

```suggestion
     * Send a log containing an error with Debug level.
```